src/libse/Common/Utilities.cs

```diff
@@ -1215,40 +1215,26 @@ public static bool StartsAndEndsWithTag(string text, string startTag, string end
                 return false;
             }
 
-            if (!text.Contains(startTag) || !text.Contains(endTag))
-            {
-                return false;
-            }
-
-            while (text.Contains("  "))
-            {
-                text = text.Replace("  ", " ");
-            }
-
-            var s1 = "- " + startTag;
-            var s2 = "-" + startTag;
-            var s3 = "- ..." + startTag;
-            var s4 = "- " + startTag + "..."; // - <i>...
```

Comment:
this was always false too... as [line 1228](https://github.com/SubtitleEdit/subtitleedit/pull/8909/files#diff-895f77470248dd0801fef50338bcd93ce27acfb2ad5de1da85485cd599e1c2cfL1228) would always match first